checkrebuild

```diff
@@ -4,51 +4,168 @@
 
 shopt -s nullglob
 shopt -s extglob
+[ -t 1 ] && declare _terminal_output=1
 
 log="$(mktemp -d)"
+touch "$log/resolved_symbols"
 trap 'rm -rf "$log"' EXIT
 
 verbose=0
-while getopts "v" OPTION; do
+while getopts "vs" OPTION; do
     case $OPTION in
-        v) verbose=1 ;;
+        s) system=1 ;;
+        v) ((verbose+=1)) ;;
         *) exit 1 ;;
     esac
 done
 
 get_unofficial_pkgs() {
-    official_repos="testing core extra community-testing community multilib-testing multilib"
-    comm -23 <(pacman -Qq | sort) <(pacman -Sl $official_repos 2>/dev/null | awk '/\[.*[[:alpha:]]+]$/ {print $2};/\[.*[[:digit:]]+]$/ {print $2}' | sort)
+    if ((system))
+    then pacman -Qq
```

Comment:
tiny: I prefer this formatting, could you please adjust?

```
if xxx; then
  stuff
fi
```

over 

```
if xxx
then stuff
fi
```

Reply:
I'm ok with both :innocent: